servers/graphql-kotlin-ktor-server/src/test/kotlin/com/expediagroup/graphql/server/ktor/GraphQLPluginTest.kt

```diff
@@ -260,11 +261,18 @@ fun Application.testGraphQLModule() {
         }
     }
     install(io.ktor.server.websocket.WebSockets)
-    install(Routing) {
+    routing {
         graphQLGetRoute()
         graphQLPostRoute()
         graphQLSubscriptionsRoute()
         graphQLSDLRoute()
         graphiQLRoute()
     }
 }
+
+private fun testModule(block: suspend io.ktor.server.testing.ApplicationTestBuilder.() -> kotlin.Unit) = testApplication {
+    environment {
+        config = ApplicationConfig(("application.conf"))
```

Comment:
Modules need to be explicitly loaded.
https://ktor.io/docs/migrating-3.html#test-module-loading